lib/active_fedora/noid/minter/db.rb

```diff
@@ -43,10 +43,13 @@ def next_id
           id
         end
 
-        def instance
-          MinterState.lock.find_by!(
-            namespace: ActiveFedora::Noid.config.namespace,
-            template: ActiveFedora::Noid.config.template
+        # @param [Boolean] lock whether the returned object (row) should be locked for update or not
+        # @return [MinterState]
+        def instance(lock = false)
+          MinterState.find_by!(
```

Comment:
Where is this style of lock documented? I tried to follow the chain and never found it. The Rails documentation seems to suggest `#lock(bool)` as maybe a preferred alternative, although I don't know what the Rails version support on that is like?